README.md

```diff
@@ -78,6 +83,8 @@ jobs:
       run: /generate-primes.sh -d prime-numbers
 
     - name: Use Prime Numbers
+      env:
+        CACHE_SKIP_SAVE: true
```

Comment:
This should be removed now that it's set on the `cache-primes` step.

Reply:
https://github.com/actions/cache/pull/489/commits/64daede5552c68991cba51f3bc0ac2bc26945a11